sig-scalability/README.md

```diff
@@ -80,7 +90,62 @@ Note that the links to display team membership will only work if you are a membe
    * 6/6
    * 5/20
 
-## Scalability/performance SLIs and SLOs
+# Details about SIG-Scalability sub-projects
+
+## Kubernetes scalability definition
+
+Defining what does it mean that "Kubernetes scales".
+This includes defining (or approving) individual performance and scalability
+related SLIs/SLOs, ensuring they are all oriented on user experience and
+consistent with each other.
+
+Measuring and publishing limits within which Kubernetes is supposed to scale
+as defined above and providing recommendations about setting clusters in
+scalable and performant ways.
+
+* [Kubernetes Scalability SLIs/SLOs](./slos/slos.md).
+
+## Kubernetes scalability governance
+
+Establishing and documenting best practises on how do design and implement
+Kubernetes features in scalable and performance way.
+Educating contributors and ensuring best practises are widely used.
+
+* [Regressions case study](./governance/scalability-regressions-case-studies.md)
+
+## Kubernetes scalability test frameworks
+
+Designing and creating frameworks to make scalability and performance testing
+of Kubernetes easy and available for all contributors.
+Different frameworks may help in different aspects of scalability testing,
+enabling making conscious tradeoffs, e.g. cost vs accuracy or real life vs
+more generalized benchmarking scenarios.
+
+* [Cluster Loader v2](https://github.com/kubernetes/perf-tests/tree/master/clusterloader2)
+* [Kubemark](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubemark)
+
+## Kubernetes scalability and performance tests and validation
+
+Ensuring that all tests necessary to validate Kubernetes scalability and
+performance exists (ideally by providing easy-to-use frameworks and working
+with SIGs to provide them), having engironment and resources to run them.
+
+Ensuring that tests are being executed according to calendar and ensuring that
+each official Kubernetes release satisfies all scalability and performance
+requirements as stateds in "Kubernetes scalability" definition.
+
+We are in progress to migrating tests to new framework, so:
```

Comment:
I'm seeing a couple of things missing here (which should be there IMO):

- https://github.com/kubernetes/community/tree/master/sig-scalability/processes (this one is in validation testing category)
- https://github.com/kubernetes/sig-release/blob/master/releases/release-*/scalability_validation_report.md (this one is in validation results category - though sadly we don't have it for newer releases)